derive/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![recursion_limit = "128"]
```

Comment:
Why is this required? I have my doubts about changing the recursion limit to this large value. Is there a better way to implement the macros?

Reply:
This is just for the quote macros, it doesn't have any effect on macros in `rustpython_vm`.

Reply:
Okay, its probably fine. The macro derive package is still a bit voodoo to me :)